rules/Php80/DocBlock/PropertyPromotionDocBlockMerger.php

```diff
@@ -51,6 +53,8 @@ public function mergePropertyAndParamDocBlocks(
 
                 $param->setAttribute(AttributeKey::COMMENTS, $mergedComments);
             }
+        } else {
+            $this->docBlockUpdater->updateRefactoredNodeWithPhpDocInfo($param);
         }
```

Comment:
above on after `$param->setAttribute(AttributeKey::COMMENTS, $mergedComments);` may probably need to use docblockupdater as well on line 54

or remove the else, direclty use:

```suggestion
        }
        
        $this->docBlockUpdater->updateRefactoredNodeWithPhpDocInfo($param);
```